### PR TITLE
[Compute] 2019-12-01 API version

### PR DESCRIPTION
**Pull request by hyonholee**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by hyonholee** [comment]
The below is expected since 'SshPublicKeyUpdateResource' is used only in a request body for update.  It is not used in a response body, so it does not need 'name', 'id', and 'type' properties.

Potential new ARM errors
========================
R2020 - RequiredPropertiesMissingInResourceModel

Model definition 'SshPublicKeyUpdateResource' must have the properties 'name', 'id' and 'type' in its hierarchy and these properties must be marked as readonly.

  at specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json:6645

**Review by markcowl — APPROVED** [review]

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json [hunk 1]
```diff
@@ -0,0 +1,10921 @@
...
+        },
+        "x-ms-examples": {
+          "Get an ssh public key.": {
+            "$ref": "./examples/GetAnSshPublicKey.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/sshPublicKeys/{sshPublicKeyName}/generateKeyPair": {
+      "post": {
```
Comment (markcowl) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json]: long-ruunning?

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json [hunk 2]
```diff
@@ -0,0 +1,10921 @@
...
+              "$ref": "#/definitions/SshPublicKeyResource"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "SshPublicKeys"
+        ],
+        "operationId": "SshPublicKeys_Delete",
```
Comment (markcowl) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json]: long-running?

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json [hunk 3]
```diff
@@ -0,0 +1,10921 @@
...
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/sshPublicKeys/{sshPublicKeyName}": {
+      "put": {
+        "tags": [
+          "SshPublicKeys"
+        ],
+        "operationId": "SshPublicKeys_Create",
```
Comment (markcowl) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json]: long-running?

**Review by markcowl — CHANGES_REQUESTED** [review]

**Comment by markcowl** [comment]
@hyonholee just verifying that none of the new operations are long-running (the examples indicate they are not)
Also
- The .Net SDK break is from a previous PR, and is in tests
- The indicated breaking change in Python is expected - the request types indicated do not have these properties.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json (hunk 1 above)
Comment (hyonholee) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json]: [5:25 PM] Nathan Kuchta
    No, they are not long running and return immediately

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json (hunk 2 above)
Comment (hyonholee) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json]: [5:25 PM] Nathan Kuchta
    No, they are not long running and return immediately

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json (hunk 3 above)
Comment (hyonholee) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json]: [5:25 PM] Nathan Kuchta
    No, they are not long running and return immediately

**Review by markcowl — APPROVED** [review]

**Pull request merged**